src/pages/Main/CategorySlide/SlideState/SlideState.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './SlideState.scss';
+
+function SlideState({ categoryPosition, position }) {
+  const match = element => element === categoryPosition;
+
+  return (
+    <li
+      className={`slideState ${position.some(match) ? 'positionNow' : null}`}
```

Comment:
false일 경우 null을 반환하게 되면 css 속성 자체가 slideState null 로 표현 될 것 같아요. 성능적인 부분은 문제가 없지만 보여지는 부분을 고려하면은 null 보다는 빈 스트링 형태로 가져가시는 게 좋을 것 같습니다!
```suggestion
      className={`slideState ${position.some(match) ? 'positionNow' : ''}`}
```